Changes

```diff
@@ -30,6 +30,10 @@ Working version
 - #8938: Extend ocamlopt option "-stop-after" to handle "scheduling" argument.
   (Greta Yorsh, review by Florian Angeletti and Sébastien Hinderer)
 
+- #8945: Fix toplevel show directive to work with constructors
```

Comment:
Shouldn't the PR itself be mentioned too?

Reply:
Thanks, I fixed it in trunk.

Reply:
Sorry about that, so typically one would refer to the PR rather than the original issue?

Reply:
One refers to both, which lets people find all the discussion about the change.